examples/intel_extension_for_pytorch/README.md

```diff
@@ -0,0 +1,179 @@
+# TorchServe with Intel® Extension for PyTorch*
+
+TorchServe can be used with Intel® Extension for PyTorch* (IPEX) to give performance boost on Intel hardware. 
+Here we show how to use TorchServe with IPEX.
+
+## Contents of this Document 
+* [Install Intel Extension for PyTorch](#install-intel-extension-for-pytorch)
+* [Serving model with Intel Extension for PyTorch](#serving-model-with-intel-extension-for-pytorch)
+* [Creating and Exporting INT8 model for IPEX](#creating-and-exporting-int8-model-for-ipex)
+* [Torchserve with Launcher](#torchserve-with-launcher)
+* [Benchmarking with Launcher](#benchmarking-with-launcher)
+
+
+## Install Intel Extension for PyTorch 
+Refer to the documentation [here](https://github.com/intel/intel-extension-for-pytorch#installation). 
+
+## Serving model with Intel Extension for PyTorch  
+After installation, all it needs to be done to use TorchServe with IPEX is to enable it in `config.properties`. 
+```
+ipex_enable=true
+```
+Once IPEX is enabled, deploying PyTorch model follows the same procedure shown [here](https://pytorch.org/serve/use_cases.html). Torchserve with IPEX can deploy any model and do inference. 
+
+## Creating and Exporting INT8 model for IPEX
+Intel Extension for PyTorch supports both eager and torchscript mode. In this section, we show how to deploy INT8 model for IPEX. 
+
+### 1. Creating a serialized file 
+First create `.pt` serialized file using IPEX INT8 inference. Here we show two examples with BERT and ResNet50. 
+
+#### BERT
+
+```
+import intel_extension_for_pytorch as ipex
+from transformers import AutoModelForSequenceClassification, AutoConfig
+import transformers
+from datasets import load_dataset
+import torch
+
+# load the model 
+config = AutoConfig.from_pretrained(
+    "bert-base-uncased", return_dict=False, torchscript=True, num_labels=2)
+model = AutoModelForSequenceClassification.from_pretrained(
+    "bert-base-uncased", config=config)
+model = model.eval()
+
+max_length = 384 
+dummy_tensor = torch.ones((1, max_length), dtype=torch.long)
+jit_inputs = (dummy_tensor, dummy_tensor, dummy_tensor)
```

Comment:
it would be helpful  for clarity to add a comment that what these dummy_tensors represent.

Reply:
sure - will include a description on this